ts/components/messagges/MessageDetailsComponent.tsx

```diff
@@ -0,0 +1,85 @@
+import * as React from "react";
+
+import { Icon, Text, View } from "native-base";
+import { connectStyle } from "native-base-shoutem-theme";
+import mapPropsToStyleNames from "native-base/src/utils/mapPropsToStyleNames";
+import { TouchableOpacity } from "react-native";
+import I18n from "../../i18n";
+import MessageDetailsInfoComponent from "./MessageDetailsInfoComponent";
+
+export type OwnProps = Readonly<{
+  serviceOrganizationName: string;
+  service: string;
+  key: string;
+  date: Date;
+  markdown: string;
+}>;
+
+type State = Readonly<{
+  isMessageDetailsInfoVisible: boolean;
+}>;
+
+export type Props = OwnProps;
+
+/**
+ * Implements a component that show the message details
+ */
+class MessageDetailsComponent extends React.Component<Props, State> {
+  constructor(props: Props) {
+    super(props);
+    this.state = { isMessageDetailsInfoVisible: false };
+  }
+  public handleDetailsInfoClick = (isVisible: boolean) => {
+    this.setState({ isMessageDetailsInfoVisible: !isVisible });
+  };
+  public renderDetailsInfoLink = () => {
+    if (!this.state.isMessageDetailsInfoVisible) {
+      return (
+        <View>
+          <Text link={true}>
+            {I18n.t("messageDetails.detailsLink.showLabel")}
+          </Text>
+          <Icon name="chevron-right" />
```

Comment:
You are using the work Icon. Use the one with io-icon-font.

Reply:
Fixed